foodplanner/lib/services/user_service.dart

```diff
@@ -0,0 +1,95 @@
+import 'dart:convert';
+import 'package:foodplanner/models/user.dart';
+import 'package:http/http.dart' as http;
+
+class UserService {
+  final String apiUrl;
+
+  UserService({required this.apiUrl});
+
+  Future<User> fetchUser() async {
+    final response = await http.get(Uri.parse('$apiUrl/api/Users/Get/1'));
+
+    if (response.statusCode == 200) {
+      return User.fromJson(jsonDecode(response.body) as Map<String, dynamic>);
+    } else {
+      throw Exception('Kunne ikke hente bruger');
+    }
+  }
```

Comment:
Vi skal sende JWT i autherization header, stort set overalt. Også for børn/forældre

Reply:
det bliver fikset i næste merge, hvor pinkode ose kommer med.